src/utilities/csv_writer.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Nov 26 17:04:32 2018
+
+CSV_auto_writer.py: Takes a header and a list of numpy arrays, and writes out a simple CSV file.
+
+@author: mmacferrin
+"""
+import numpy
+
+def write_csv(data, csvname, header=None, verbose=False, print_help=False):
```

Comment:
The same as the previous one. Pandas dataframes allows categorizing data in a column based approach.

Reply:
I'm not against it. I'm dating myself... I'd been using a homegrown simple solution like this since before pandas was mainstream. Feel free to change the implementation to use pandas instead if preferred.

Reply:
I see :) I was really curious, because it seems as a lot of work!